code/datums/jobs/job/shipside.dm

```diff
@@ -662,6 +664,22 @@ A happy ship is a well-functioning ship."})
 You have final authority over the medical department, medications, and treatments.
 Make sure that the doctors and nurses are doing their jobs and keeping the marines healthy and strong."})
 
+/datum/job/terragov/medical/professor/after_spawn(mob/living/carbon/new_mob, mob/user, latejoin = FALSE)
+	. = ..()
+	if(!ishuman(new_mob))
+		return
+	var/mob/living/carbon/human/new_human = new_mob
+	var/playtime_mins = user?.client?.get_exp(title)
+	if(!playtime_mins || playtime_mins < 1 )
+		return
+	switch(playtime_mins)
+		if(0 to 1500) // starting
+			new_human.wear_id.paygrade = "O3"
+		if(1501 to 3000) // 25 hrs
+			new_human.wear_id.paygrade = "O4"
+		if(3001 to INFINITY) // 50 hrs
+			new_human.wear_id.paygrade = "O5"
```

Comment:
Commander doesnt fit make medical ranks 
Ie for example

Dr.
prof.
etc

Reply:
So the CMO goes through Navy ranks and then after a certain amount of service is suddenly not a naval officer despite being a medical office on board a Navy ship? I kind of see what you mean by "Commander doesn't fit" _I guess_, but it's a naval officer, Commander is a Navy officer rank. Medical officers also tend to be slightly higher rank/more seasoned by virtue of medical training adding on years of service and age.